fab/environments.yml

```diff
@@ -111,6 +115,8 @@ production:
         concurrency: 8
       ucr_queue:
         concurrency: 4
+      email_queue:
+        concurrency: 4
```

Comment:
ok, i was just going by what you did for UCR, but that makes sense, I'll update that number now